content/en/synthetics/ci.md

```diff
@@ -257,74 +368,37 @@ Then, run:
 npm run datadog-ci-synthetics
 ```
 
+**Note**: If you are launching your tests with a custom global configuration file, append the command associated to your `datadog-ci-synthetics` script with `--config <PATH_TO_GLOBAL_CONFIG_FILE`. 
+
+{{% /tab %}}
+{{% tab "System wide" %}}
+
+```
+datadog-ci synthetics run-tests
+```
```

Comment:
This part feels weird, it's more you can use that command, if you installed it globally but still with `yarn` or `npm` so it can in both tabs (also there's nothing specific to us here, it's just how node modules work)